monolith/views/auth.py

```diff
@@ -39,14 +36,14 @@ def create_user():
     if not current_user.is_anonymous:
         return redirect("/", code=302)
```

Comment:
Maybe add message to help testing?

Reply:
IMHO, this is more of a quality of life feature, not something we really have to test.

I would stick to testing the major features offered by the website, and ignore the small details